src/newrelic.coffee

```diff
@@ -34,16 +35,24 @@
 
 gist = require 'quick-gist'
 moment = require 'moment'
+diff = require 'fast-array-diff'
 
 plugin = (robot) ->
   apiKey = process.env.HUBOT_NEWRELIC_API_KEY
   apiHost = process.env.HUBOT_NEWRELIC_API_HOST or 'api.newrelic.com'
+  room = process.env.HUBOT_NEWRELIC_ALERT_ROOM
+  throw new Error('HUBOT_NEWRELIC_API_KEY is not set') unless apiKey
+  throw new Error('HUBOT_NEWRELIC_ALERT_ROOM is not set') unless room
```

Comment:
Instead of completely killing the bot if this script is misconfigured, just abort this script and print an error to console.

```
return robot.logger.error "Please provide your New Relic API key at HUBOT_NEWRELIC_API_KEY" unless apiKey
return robot.logger.error "Please specify a room to report New Relic notifications to at HUBOT_NEWRELIC_ALERT_ROOM" unless room
```